### PR TITLE
Autorefocus on "Open current map"

### PR DESCRIPTION
**Pull request by BannZay**
The editor refocuses to the place that the user looked at in the game
It will be easier and faster to find things you are looking for

### CODE REVIEW

File: src/game/editor/editor.cpp [hunk 1]
```diff
@@ -207,7 +209,7 @@ void CLayerGroup::Render()
 
 void CLayerGroup::AddLayer(CLayer *l)
 {
-	m_pMap->m_Modified = true;
+ 	m_pMap->m_Modified = true;
```
Comment (def-) [inline src/game/editor/editor.cpp:212]: what's this? looks like a space whereas we use tabs

File: src/game/editor/editor.cpp [hunk 2]
```diff
@@ -6287,6 +6289,18 @@ void CEditor::UpdateAndRender()
 	Input()->Clear();
 }
 
+void CEditor::LoadCurrentMap()
```
Comment (def-) [inline src/game/editor/editor.cpp:6292]: Why did you move this away from io.cpp?

File: src/game/editor/editor.cpp [hunk 3]
```diff
@@ -6325,3 +6339,4 @@ void CEditorMap::MakeTuneLayer(CLayer *pLayer)
 	m_pTuneLayer->m_pEditor = m_pEditor;
 	m_pTuneLayer->m_Texture = m_pEditor->m_TuneTexture;
 }
+
```
Comment (def-) [inline src/game/editor/editor.cpp:6342]: another whitespace

File: src/game/editor/editor.cpp (hunk 2 above)
Comment (BannZay) [reply src/game/editor/editor.cpp:6292]: I could modified io.cpp LoadCurrentMap method, but, seems, editor camera manipulation code should reside at editor.cpp 

So I created new method for it at editor.cpp with call to LoadCurrentMap from io.cpp. But io.cpp version was referenced by newly created method only, so I just extracted its body to new method and removed it from io.cpp

Please, correct me if I am wrong at any point. I will be happy to make it "right" or at least match your vision of it

**Comment by heinrich5991** [comment]
Looks good. Please sort the inputs and adhere to the variable naming scheme. :)

**Comment by heinrich5991** [comment]
The star for pointer types goes next to the variable name in Teeworlds code, pointer variables get a `p` prefix: `CGameClient *pGameClient`.

**Comment by heinrich5991** [comment]
Thanks!

bors r+
